.github/workflows/pr-prefix.yml

```
@@ -0,0 +1,19 @@
+name: check-pr-title-prefix
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  check-pr-prefix:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR title prefix
+        run: |
+          if echo "${{ github.event.pull_request.title }}" | tr ':' '\n' | head -n 1 | grep -qE 'feat|fix|docs|refactor|ci|chore(\([^)]+\)|$)';
+          then
+            echo "PR title is valid"
+          else
+            echo "PR title is invalid"
```

Let's elaborate what to do when new devs see this error:
```suggestion
            echo "PR title is invalid."
            echo "Use the title prefixes like:"
            echo "  feat: (for new features and functionality)"
            echo "  fix: (for bug fixes and revisions on how things work)"
            echo "  docs: (for docs, docstring, and comment changes)"
            echo "  refactor: (for refactoring and revisions on how things are related)"
            echo "  ci: (for changes related to CI/CD workflows)"
            echo "  chore(...): (for changes related to repo/build configs, dependencies, etc.)"
            echo "following the conventional commit style."
```